src/test/java/no/brreg/regnskap/integration/RegnskapApiTest.java

```diff
@@ -42,7 +43,7 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-
+@Disabled("Disabled until TestContainer support have been added to Jenkins slave")
```

Comment:
Siden disse testene krever TestContainers og går mot en database, så er dette vel det vi på åpne data kaller EnhetsIntegrasjonstester. Slike tester har vi da navngitt -IT eller -IntegrationTest. Så benyttes profilen i maven til å bestemme hvilke tester som skal kjøres. Det er mulig å gjør det samme her, og da ikke bruke all-tests, men f.eks. dev som profile inn i bygget, inntil disse testene har blitt skrevet om. Når testene er disabled, så kjøres de jo ikke lokalt heller, selv med all-tests.

Reply:
Har merget inn Michaels endringer, så nå er testene reaktivert igjen. Har endret navn på testene til *IT i commit bd2ecb87